src/main/java/seedu/address/logic/parser/AddEventParser.java

```diff
@@ -94,6 +94,17 @@ private static ParseResult parseArguments(String str) throws IllegalValueExcepti
             result.startTime = args.remove(0);
         }
 
+        // Check Keyword "to"
+        if (args.isEmpty()) {
+            throw new IllegalValueException("expected ending time or ending date");
+        }
+        if (!isKeywordTo(args.get(0))) {
+            throw new IllegalValueException("expected keyword \"to\"");
+        }
+        if (isKeywordTo(args.get(0))) {
+            args.remove(0);
+        }
+
```

Comment:
This could be easily simplified to
```java
if (args.get(0).equals("to")) {
    args.remove(0);
} else {
    throw new IllegalValueException("...");
}
```
no?